src/targets/vendor/assets/service-worker.js

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
What is this file for?

Reply:
Is needed for an correct installation of SW, which is required to allow the user to refresh their PWA